tensorflow/core/kernels/cwise_ops.h

```diff
@@ -1318,6 +1318,13 @@ struct UnaryFunctor {
                   typename Functor::tin_type in);
 };
 
+template <typename Device, typename Functor>
+struct UnaryFunctorWithArg {
+  // Computes on device "d": out[i] = Functor(in[i])
+  void operator()(const Device& d, typename Functor::tout_type out,
+                  typename Functor::tin_type in, float val);
```

Comment:
Can we make this more generic, rather than requiring the arg to be of type `float`?

Reply:
yes, will do